features/bevragen/zoek-met-geslachtsnaam-en-geboortedatum/fout-cases.feature

```diff
@@ -226,7 +62,7 @@ Rule: Geslachtsnaam en geboortedatum zijn verplichte parameters
     | required | geboortedatum | Parameter is verplicht. |
 
   @fout-case
-  Scenario: Zoek met leeg geslachtsnaam en leeg geboortedatum
+  Scenario: Een lege string is opgegeven als geslachtsnaam en geboortedatum waarde
```

Comment:
Is de waarde 'Parameter is verplicht.' voor reason logisch in deze situatie. In principe worden de parameters geslachtsnaam en geboortedatum immers meegegeven maar met een verkeerde waarde.
De reasons 'Waarde is geen geldige datum.' en 'Waarde voldoet niet aan patroon ^[a-zA-Z0-9À-ž \.\-\']{1,200}$\|^[a-zA-Z0-9À-ž \.\-\']{3,199}\*{1}$.' lijken mij logischer.

Reply:
Delen de anderen deze mening? Voor mij maakt het niet uit. Je zou zelfs zo ver kunnen gaan dat je zegt dat de opgegeven waarde niet geldig zijn voor alle gevallen. Dat scheelt ook in aantal verschillende codes en omschrijvingen

Reply:
Ik denk dat het niet uitmaakt of je bij een lege string required of pattern opgeeft. Een afnemer weet in beide gevallen wat er moet gebeuren (een waarde opgeven die wel voldoet aan het pattern of format date). 

Wanneer de parameter helemaal niet gestuurd is, is het geven van een pattern-fout vreemd, mogelijk verwarrend. Kan de clientdeveloper op het verkeerde been zetten.

@MelvLee  kan jij in je code het verschil zien tussen een parameter niet krijgen, een parameter als null krijgen en een parameter als "" krijgen?